locales/en.json

```diff
@@ -4,7 +4,7 @@
     "HeroMessage": "Crawl, walk, run."
   },
 	"Deals": {
-		"CitiesTitle": "Destinations you can walk to.",
+		"CitiesTitle": "Destinations you can walk to. April Fools!",
```

Comment:
Please spell Destinations correctly.  Other than that, this looks good!

Reply:
Sure I can fix that.